src/main/docker/logback.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright 2017-2020 Micro Focus or one of its affiliates.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+         http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<configuration>  
+
+    <appender name="STDOUT" class="com.github.cafapi.logging.tomcat.juli.CAFTomcatConsoleAppender">
+        <encoder> 		 
+            <pattern>[%date{HH:mm:ss.SSS,UTC}Z %thread %-5level -            -   ] %logger{30} : %msg%n</pattern>
+        </encoder>
```

Comment:
Shouldn't you be specifying the character set in this one like you are in the other one?

Reply:
Updated and will raise New PR from another branch.